store/mock/mockSPDB.go

```diff
@@ -0,0 +1,184 @@
+package mock
```

Comment:
`mockgen -source=store/sqldb/database.go -destination=store/sqldb/database_mock.go -package=sqldb` has existed in the makefile.